From 6e069e206ccaa20e60d69929c875a14d83ed2e87 Mon Sep 17 00:00:00 2001
From: Hai-Ying Wang <Haiying.Wang@freescale.com>
Date: Thu, 22 Nov 2012 00:19:42 +0000
Subject: [PATCH 176/227] fsl_qman: increase the qman ccsr reg size to 0x2000

QManV3 uses larger CCSR reg map.

Signed-off-by: Haiying Wang <Haiying.Wang@freescale.com>
Signed-off-by: Andy Fleming <afleming@freescale.com>
[Kevin: Original patch taken from fsl sdk 1.3.1
QorIQ-SDK-V1.3.1-SOURCE-20121220-yocto.iso.]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 arch/powerpc/boot/dts/fsl/qoriq-qman1.dtsi |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/powerpc/boot/dts/fsl/qoriq-qman1.dtsi b/arch/powerpc/boot/dts/fsl/qoriq-qman1.dtsi
index a9d3991..8edd2c0 100644
--- a/arch/powerpc/boot/dts/fsl/qoriq-qman1.dtsi
+++ b/arch/powerpc/boot/dts/fsl/qoriq-qman1.dtsi
@@ -34,6 +34,6 @@
 
 qman: qman@318000 {
 	compatible = "fsl,qman";
-	reg = <0x318000 0x1000>;
+	reg = <0x318000 0x2000>;
 	interrupts = <16 2 1 3>;
 };
-- 
1.7.9.7

